docs/concepts/storage/volumes.md

```diff
@@ -171,6 +172,43 @@ writers simultaneously.
 
 See the [CephFS example](https://github.com/kubernetes/examples/tree/{{page.githubbranch}}/staging/volumes/cephfs/) for more details.
 
+### configMap
+
+The [`configMap`](/docs/tasks/configure-pod-container/configmap/) resource
+provides a way to inject configuration data into Pods.
+The data stored in a `ConfigMap` object can be referenced in a `configMap` type
+of volume and then consumed by containerized applications running in a Pod.
+
+When referencing a `configMap` object, you can simply provide its name in the
+volume to reference it. You can also customize the path to use for a specific
+entry in the ConfigMap.
+For example:
```

Comment:
Replace "For example:" with:
>  For example, to mount the `log-config` ConfigMap onto a Pod called`configmap-pod`, you might use the YAML below:

This makes it clearer to the reader what they're about to see.